src/sidebar/util/test/fetch-config-test.js

```diff
@@ -23,61 +27,58 @@ describe('sidebar.util.fetch-config', () => {
     fakeJsonRpc.call.throws(new Error('call() response not set'));
 
     // Setup fake window hierarchy.
-    const fakeTopWindow = { parent: null, top: null };
+    fakeTopWindow = { parent: null, top: null, index: 2 };
```

Comment:
It wasn’t immediately clear to me what the `index` property was since real windows don’t have this. I assume this is just for use in test asserts? Maybe call it `testId` or something similar to make that more obvious?

Reply:
correct, index was a terrible name